spec/searchers/responses_searcher_spec.rb

```diff
@@ -56,6 +71,23 @@
     end
   end
 
+  describe "submitter qualifier" do
+    let!(:u1) { create(:user, name: "u1") }
+    let!(:u2) { create(:user, name: "u2 name") }
+    let!(:u3) { create(:user, name: "u3") }
+    let!(:r1) { create(:response, user: u1) }
+    let!(:r2) { create(:response, user: u2) }
+
+    it "should work" do
+      expect(search(%(submitter:#{u1.name}))).to contain_exactly(r1)
+
+      expect(searcher(%(submitter:#{u1.name}))).to have_filter_data(submitters: items_by_name_and_id(u1), advanced_text: "")
```

Comment:
I would consider getting rid of `items_by_name_and_id`. It's not shortening things much over `{id: u1.id, name: "u2 name"}` and it adds a layer of abstraction that makes the test harder to read. The reviewer test is so nice because it doesn't have this.